lib/chef/chef_fs/file_system/chef_server/chef_server_root_dir.rb

```diff
@@ -92,6 +94,7 @@ def initialize(root_name, chef_config, options = {})
           attr_reader :repo_mode
           attr_reader :cookbook_version
           attr_reader :versioned_cookbooks
+          attr_reader :policy_document_native_api
```

Comment:
Seems like this is unused? I don't think it would affect the behavior of the server anyway, just the client.